superset/migrations/env.py

```diff
@@ -117,6 +130,7 @@ def process_revision_directives(  # pylint: disable=redefined-outer-name, unused
     try:
         with context.begin_transaction():
             context.run_migrations()
+            print_duration(start_time)
```

Comment:
Should this be outside the transaction context? I assume the context manager here is going to commit the transaction on exit, so it's not really done until that happens.

Reply:
Good point